frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -249,17 +239,32 @@ const TestCardForm = ({
   };
 
   const validateTestResults = () => {
-    if (deviceSupportsMultiplex) {
-      const multiplexResults = convertFromMultiplexResultInputs(
-        state.testResults
-      );
-      return validateMultiplexResultState(
-        multiplexResults,
-        state.deviceId,
-        devicesMap
+    const supportedDiseaseNames =
+      state.devicesMap
+        .get(state.deviceId)
+        ?.supportedDiseaseTestPerformed.map(
+          (supportedTest) => supportedTest.supportedDisease.name
+        ) ?? [];
+    let validResults: MultiplexResultInput[] = [];
+    state.testResults.forEach((result) => {
+      // check that result's disease name is supported by the current device
+      const hasSupportedDiseaseName = supportedDiseaseNames.some(
+        (diseaseName) => diseaseName === result.diseaseName
       );
+      const isFilled =
+        result.testResult === TEST_RESULTS.POSITIVE ||
+        result.testResult === TEST_RESULTS.NEGATIVE ||
+        result.testResult === TEST_RESULTS.UNDETERMINED;
```

Comment:
could we just use `result.testResult !== TEST_RESULTS.UNKNOWN` here? [Reading the enum definition ](https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/testResults/constants.ts) those two seem to be equivalent

Reply:
The `MultiplexResultInput` type allows test result to be undefined, so I'd lean toward keeping these to be safe. Also realized from this that the test results aren't being explicitly set as `TEST_RESULTS.UNKNOWN` in the form state so I'll update that